build-system/compile/compile.js

```diff
@@ -58,6 +62,86 @@ function convertPathsToTmpRoot(paths) {
   return paths.map(path => path.replace(/^(!?)(.*)$/, `$1${SRC_TEMP_DIR}/$2`));
 }
 
+function loadSourceMap(file) {
```

Comment:
Can you document this function?

Reply:
Updated with comments and moved to standalone location.